mutable_tree.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/cosmos/iavl/internal/logger"
 )
 
+var commitGap uint64 = 10000000
```

Comment:
let's add a comment here

Reply:
had add a comment

Reply:
have you tried different numbers to see results?

Reply:
> have you tried different numbers to see results?

not yet. maybe I can use a different value to comare with this. do you have any sugguestion? maybe 5,000,000?

Reply:
yea lets try it

Reply:
with 5m, when got lower memory usage and faster speed ;P
<img width="597" alt="image" src="https://user-images.githubusercontent.com/5326600/189021628-4f729420-9bd0-44b5-9a75-80eb8e8ef75c.png">

Reply:
amazing, lets run with it

Reply:
ok, had change the commitGap to 5000000, and remove the timing-gc related code.